src/covid19sim/simulator.py

```diff
@@ -1339,7 +1458,10 @@ def at(self, location, city, duration):
                         infectee.compute_covid_properties()
 
                         infector.n_infectious_contacts += 1
-                        Event.log_exposed(infectee, infector, self.env.timestamp)
+
+                        if self.conf.get('COLLECT_LOGS'):
+                            Event.log_exposed(h, self, self.env.timestamp)
```

Comment:
This looks like a bug and should be `Event.log_exposed(infectee, infector, self.env.timestamp)` based on lines 1443 - 1445

Reply:
yes its a bug!!

Reply:
fixed